### PR TITLE
[WIP] Update rebuild form layout and add static field for reset password

### LINKED ISSUE #943: Reset root password should have no disk option when there is only one disk
screen shot 2017-01-09 at 4 34 45 pm

### PR DESCRIPTION
**Pull request by mparke**
Opening a wip since this touches multiple issues:

closes #1065 
closes #943

### CODE REVIEW

**Comment by mparke** [comment]
oof, formatting got reset here, will see if easy fix, if not will comment actual changes.

**Comment by mparke** [comment]
On second thought I messed this up, closing and fixing.

File: scss/components/form/form.scss [hunk 1]
```diff
@@ -37,6 +37,10 @@ form {
     display: inline-block;
 }
 
+.StaticFormField {
+    line-height: $input-height;
+}
```
Comment (eatonphil) [inline scss/components/form/form.scss]: Could you drop line 67 stuff too then?:

```

.input-line-height {
    line-height: $input-height;
}
```

Comment (eatonphil) [inline scss/components/form/form.scss]: (And replace usage of that class)

**Pull request closed without merging**